### DISCUSSION TITLE
Update example to Svelte 5

### DISCUSSION THREAD

**Discussion by wlto** [vercel/turborepo#9544] [Ideas]
### Goals

1. Bump SvelteKit template up to use Svelte 5.
2. Update template's ESLint to work nicely with Svelte 5 while not conflicting with other toolings.

### Non-goals

1.
2.
3.

### Background

I am trying to set up a monorepo within my team that should be using Svelte 5. Turborepo is a good tool but the template provided with the `with-svelte` option is current using Svelte 3. This would allow us to use [runes](https://svelte.dev/docs/svelte/what-are-runes).

### Proposal

I'm interested in contributing. However, I'm not too familiar with build tools and new to monorepo. So might need someone to point me in the right direction.

**Comment by anthonyshew** [comment]
Turborepo supports Svelte 5 (and any other frameworks). You can use it in a Turborepo today.

That said, yes, the example could use an update.

**Reply by wlto** [reply]
Hey @anthonyshew ! Thank you for your reply.
Sounds good. But yeah when I upgraded Svelte to version 5 using `npx sv create`, it seems like a couple of things are not working, including ESLint. I guess I just need to figure out a way to make it work. Or perhaps I just did something wrong.

I'll look into this tomorrow and report back if I find anything or if I have an updated example for review.

**Reply by wlto** [reply]
Okay I think I just did something wrong.
Upgraded these two packages and everything works well again:
- `svelte`: `^4.2.12` to `^5`
- `@sveltejs/vite-plugin-svelte`: `^3.0.2` to `^4.0.0`

I'm open to create a PR for this. Is there any test I should run beforehand? I took a look at the [Contributing](https://github.com/vercel/turborepo) doc and just not sure if I need to run any test before making a PR.

**Reply by anthonyshew** [reply]
Our CI has tests for examples and I'll be sure to review, so submitting the fixes into https://github.com/vercel/turborepo/tree/main/examples/with-svelte will be the right start!

**Reply by anthonyshew** [reply]
Will also update our Contributing doc with some improvements.
